docs/en/engines/database-engines/datalake.md

```diff
@@ -45,23 +45,39 @@ catalog_type,
-| `warehouse`             | The warehouse/database name to use in the catalog.                        |
-| `catalog_credential`    | Authentication credential for the catalog (e.g., API key or token)        |
-| `auth_header`           | Custom HTTP header for authentication with the catalog service            |
-| `auth_scope`            | OAuth2 scope for authentication (if using OAuth)                          |
-| `storage_endpoint`      | Endpoint URL for the underlying storage                                   |
-| `oauth_server_uri`      | URI of the OAuth2 authorization server for authentication                 |
-| `vended_credentials`    | Boolean indicating whether to use vended credentials (AWS-specific)       |
-| `aws_access_key_id`     | AWS access key ID for S3/Glue access (if not using vended credentials)    |
-| `aws_secret_access_key` | AWS secret access key for S3/Glue access (if not using vended credentials) |
-| `region`                | AWS region for the service (e.g., `us-east-1`)                             |
+| Setting                 | Description                                                                             |
+|-------------------------|-----------------------------------------------------------------------------------------|
+| `catalog_type`          | Type of catalog: `glue`, `unity` (Delta), `rest` (Iceberg), `hive`, `onelake` (Iceberg) |
+| `warehouse`             | The warehouse/database name to use in the catalog.                                      |
+| `catalog_credential`    | Authentication credential for the catalog (e.g., API key or token)                      |
+| `auth_header`           | Custom HTTP header for authentication with the catalog service                          |
+| `auth_scope`            | OAuth2 scope for authentication (if using OAuth)                                        |
+| `storage_endpoint`      | Endpoint URL for the underlying storage                                                 |
+| `oauth_server_uri`      | URI of the OAuth2 authorization server for authentication                               |
+| `vended_credentials`    | Boolean indicating whether to use vended credentials (AWS-specific)                     |
+| `aws_access_key_id`     | AWS access key ID for S3/Glue access (if not using vended credentials)                  |
+| `aws_secret_access_key` | AWS secret access key for S3/Glue access (if not using vended credentials)              |
+| `region`                | AWS region for the service (e.g., `us-east-1`)                                          |
 
 ## Examples {#examples}
 
-See below pages for examples of using the `DataLakeCatalog` engine:
+See below sections for examples of using the `DataLakeCatalog` engine:
 
 * [Unity Catalog](/use-cases/data-lake/unity-catalog)
 * [Glue Catalog](/use-cases/data-lake/glue-catalog)
+* OneLake Catalog
```

Comment:
@Blargian : I have just added a simple usage example. We can elaborate and move to separate page when we add docs for other catalogs too. Hope this is fine.